.circleci/config-2_1.yml

```diff
@@ -1102,3 +1207,174 @@ commands:
       - store_artifacts:
           path: ~/cassandra-dtest/logs
           destination: dtest_<<parameters.file_tag>>_logs
+
+  run_repeated_utest:
+    steps:
+      - run:
+          name: Run repeated utest
+          no_output_timeout: 15m
+          command: |
+            if [ "$REPEATED_UTEST_CLASS" == "<nil>" ]; then
```

Comment:
Where does this `<nil>`come from?

Reply:
That's what CircleCI puts on empty env vars. I think that the `nil` comes from Go.

Reply:
sgtm thx.